internal/pkg/utils/strhelper/strhelper.go

```diff
@@ -69,3 +69,13 @@ func FirstLower(str string) string {
 func FirstUpper(str string) string {
 	return strings.ToUpper(string(str[0])) + str[1:]
 }
+
+// StripHtmlComments replaces HTML comments with empty lines.
+func StripHtmlComments(str string) string {
+	return regexpcache.
+		MustCompile("(?s)<!--(.*?)-->").
```

Comment:
`?s` - `.` match `\n`
`.*?` - [lazy/ungreedy](https://stackoverflow.com/questions/2301285/what-do-lazy-and-greedy-mean-in-the-context-of-regular-expressions), means match shortest possible string.